tools/delete.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/sh
```

Comment:
Verify this is the intended result:
echo '12aaa34aaa56' > xx
./delete.sh 'aa' 'aa' < xx > yy
cat yy
12a56

Reply:
At least, that matches the behavior I intended for it: the second match starts where the first match ended, so they can't overlap.

Reply:
delete str str deletes the first occurrence of str now, a corner case with simple semantics for which I can envision applications.

Reply:
ack